ui/components/component-library/badge-wrapper/badge-wrapper.types.ts

```diff
@@ -46,6 +46,9 @@ export interface BadgeWrapperStyleUtilityProps extends StyleUtilityProps {
     right?: number | string;
     bottom?: number | string;
     left?: number | string;
+    transform?: string;
+    width?: string;
+    height?: string;
```

Comment:
Instead of adding these additional prop types, can we add a custom class name to the badge container via the `badgeContainerProps` object? I noticed that we could make some improvements to the `BadgeWrapper` component on line `37`:

```jsx
<Box
  {...badgeContainerProps} // won't overwrite className prop if `badgeContainerProps.className` exists
  className={classnames(
    'mm-badge-wrapper__badge-container',
    {
      [`mm-badge-wrapper__badge-container--${anchorElementShape}-${position}`]: !positionObj,
    },
    badgeContainerProps?.className || '', // custom class name which can contain the transform styles
  )}
  style={{ ...positionObj }}
>
```

Reply:
Done: [c41aa39](https://github.com/MetaMask/metamask-extension/pull/23697/commits/c41aa39552509a3bcf101ca7ac9864d653777ddf)